lesson-2/web-store/src/pages/user_page.js

```diff
@@ -0,0 +1,45 @@
+import React, {Component} from 'react';
+import {ShopItem} from './shop_item';
+import {Logout} from './../components/shared/logout';
+import {Orders} from './../components/shared/orders_button.js';
+
+
+export class User extends Component {
+  render() {
+    const items = this.props.items;
+    const inCart = this.props.cart;
+    const returnListOfItems = (items) => {
+      return items.map((item ) => {
+        let inBasket = false;
+        if(inCart.indexOf(item.id) !== -1){
+          inBasket = true;
+        }
+        const addItemToCart = () => {
+          this.props.addItemToCart(item.id);
+        }
+        return <ShopItem key={item.id} item={item} inBasket={inBasket} addItemToCart={addItemToCart} ></ShopItem>
```

Comment:
there a lot of properties it's better to make it multiline. Or just simply use a prettier